worlds/hk/__init__.py

```diff
@@ -252,21 +252,21 @@ def _add(item_name: str, location_name: str):
             if item_name in junk_replace:
                 item_name = self.get_filler_item_name()
 
-            item = self.create_item(item_name)
```

Comment:
Would probably be more elegant to do something like this pseudocode, `create_event() if vanilla else create_item()` to avoid the repetition, possibly some additional refactoring could be needed there too not sure

Reply:
looks like `item = self.create_item(item_name) if not vanilla or location_name == "Start" else self.create_event(item_name)` works (because otherwise all of starting inventory becomes events and the player never recieves them), then we can return the rest to just `item` is that what you were thinking?

Reply:
Yeah something like that